.jenkinsci/docker-pull-or-build.groovy

```diff
@@ -19,65 +19,59 @@ def buildOptionsString(options) {
 
 def dockerPullOrUpdate(imageName, currentDockerfileURL, previousDockerfileURL, referenceDockerfileURL, buildOptions=null) {
   buildOptions = buildOptionsString(buildOptions)
+  def uploadExitCode = 0
   // GIT_PREVIOUS_COMMIT is null for first PR build
```

Comment:
I do not think you can simply remove this block. You should check whether `previousDockerfileURL` is `null`. Otherwise, you end up querying URL with literal `null` string in it.

Reply:
@bakhtin I think we can as `previousDockerfileURL` passed to the `dockerPullOrBuild()` method is built from `pCommit` variable, which cannot be `null`, otherwise it's out of scope of method `dockerPullOrBuild()` and should be fixed in the `.jenkinsci/previous-commit.groovy` `previousCommitOrCurrent()` method